### DISCUSSION TITLE
Error in determining project root

### DISCUSSION THREAD

**Discussion by amortsell** [Azure/logicapps#1025] [Help Needed]
I'm pretty new to logic apps standard and was recently introduced in a project where standard logic apps is used for all workflows. The logic apps are deployed to different resource groups and the code is structured like this
````
├── rg1
│   ├── logic
│   │   ├── logic-rg1
│   │   │   ├── .vsocde
│   │   │   │   ├── extensions.json
│   │   │   │   ├── launch.json
│   │   │   │   ├── settings.json
│   │   │   │   └── tasks.json
│   │   │   ├── deploy
│   │   │   │   ├── <file structure for pipelines and bicep deploy scripts>
│   │   │   ├── source
│   │   │   │   ├── logic-app-1
│   │   │   │   │   └── workflow.json
├── rg2
│   ├── logic
│   │   ├── logic-rg2
│   │   │   ├── .vsocde
│   │   │   │   ├── extensions.json
│   │   │   │   ├── launch.json
│   │   │   │   ├── settings.json
│   │   │   │   └── tasks.json
│   │   │   ├── deploy
│   │   │   │   ├── <file structure for pipelines and bicep deploy scripts>
│   │   │   ├── source
│   │   │   │   ├── logic-app-1
│   │   │   │       └── workflow.json
│   │   │   │   ├── logic-app-2
│   │   │   │       └── workflow.json
│   │   │   │   ├── .funcignore
│   │   │   │   ├── .gitignore
│   │   │   │   ├── connections.json
│   │   │   │   ├── host.json
│   │   │   │   └── parameters.json
│   ├── func
│   │   │   ├── <file structure for function app projects>
├── dist (or build)
├── node_modules
├── package.json
├── package-lock.json
└── .gitignore 
````

In the logic-rg1 case, I can open the folder rg1\logic\logic-rg1 in VS Code then right-click the workflow.json file and select open in designer, and everything works as expected but in the second case, when I open the folder rg2\logic\logic-rg2, where I have two workflows in the same project, I get the error message "Error in determining project root. Please confirm project structure is correct" when I try o open the designer. I have verified that the files in the .vscode folder match between the two projects and have tested different settings for "azureLogicAppsStandard.projectSubpath", which I have found useful in other cases, but I haven't found anything that works in this case. However, in the second case, I get the same error if I remove the logic-app-2 folder from the structure and try to open logic-app-1\workflow.json. Does anyone have any idea on what is going on here, and what I could do to solve this issue?

**Comment by arnaudluti** [comment]
Same here
